pkg/connector/connector.go

```diff
@@ -57,9 +62,10 @@ func (c *Calendly) Validate(ctx context.Context) (annotations.Annotations, error
 }
 
 // New returns a new instance of the connector.
-func New(ctx context.Context, token string) (*Calendly, error) {
+func New(ctx context.Context, cfg *viper.Viper) (*Calendly, error) {
 	var (
 		httpClient *http.Client
+		token      = cfg.GetString(Token)
 		err        error
 	)
 	if token == "" {
```

Comment:
is it ok to delete this condition if `token` is supposed to be required?

Reply:
in some cases, when the connector is running in service mode I think this condition can be trigger